### ISSUE TITLE
Python 3 support

### ISSUE THREAD

**Issue by kmike** [scrapy-plugins/scrapy-djangoitem#10]
Scrapy 1.1.0rc1 supports Python 3, so it'd be nice to have Python 3 support for scrapy-djangoitem as well. Contributions are welcome!

**Comment by jschilling1** [comment]
The following works in Python 2 and Python 3, can someone test /  commit please?

```
 from django.core.exceptions import ValidationError
 from scrapy.item import Field, Item, ItemMeta
-
+from six import with_metaclass


-class DjangoItem(Item):
-
-    __metaclass__ = DjangoItemMeta
+class DjangoItem(with_metaclass(DjangoItemMeta, Item)):

```

**Comment by holgerd77** [comment]
Hi @kmike, I added the test environment for Python 3.4 and 3.5 and updated the tests to pass on Django 1.9, this should now be solid.

**Comment by kmike** [comment]
Thanks @jschilling1 and @holgerd77! Fixed by #11.
